### DISCUSSION TITLE
cmd + K to open search

### DISCUSSION THREAD

**Discussion by jukremer** [jpanther/congo#520] [Ideas]
Many websites have implemented this so far. I think Congo should have this feature too.

**Comment by stereobooster** [comment]
Kongo's hotkey for search is <kbd>/</kbd>. The same as github

**Reply by jukremer** [reply]
I didn’t know that, where would I find that?

**Reply by stereobooster** [reply]
Fair point - I found it in source code (because was debugging other issue)

**Reply by jpanther** [reply]
It does also appear in the tooltip on hover :)
